src/a11y-dialog.ts

```diff
@@ -56,7 +56,8 @@ export default class A11yDialog {
 
     // Keep a reference to the currently focused element to be able to restore
     // it later
-    this.previouslyFocused = getActiveElement() as HTMLElement
+    this.previouslyFocused =
+      (getActiveElement() as HTMLElement) || document.activeElement
```

Comment:
I feel like `|| document.activeElement` should be done in `getActiveElement()`. After all, it’s in the name. 😛

Reply:
Fair enough!